### PR TITLE
Code Style: add `binary_operator_spaces` rule

### PR DESCRIPTION
**Pull request by KennedyTedesco**

### CODE REVIEW

File: src/Model/Modifier.php [hunk 1]
```diff
@@ -8,16 +8,16 @@
...
+    public const NONE = 0b000000000000;
+    public const BOLD = 0b000000000001;
+    public const DIM = 0b000000000010;
+    public const ITALIC = 0b000000000100;
+    public const UNDERLINED = 0b000000001000;
+    public const SLOWBLINK = 0b000000010000;
+    public const RAPIDBLINK = 0b000000100000;
+    public const REVERSED = 0b000001000000;
+    public const HIDDEN = 0b000010000000;
+    public const CROSSEDOUT = 0b000100000000;
```
[the remark this answers is not in the document]
Comment (dantleech) [reply src/Model/Modifier.php:20]: would be happy to require consecuitive assignments to be aligned

[the remark this answers is not in the document]
Comment (KennedyTedesco) [reply src/Model/Modifier.php:20]: @dantleech They don't have a specific rule for class constants, so a configuration like that will force spaces after every new variable assignment. So, I think we can manually take care of those.

But they do have this one:

```php
match ($alignment) {
    VerticalAlignment::Bottom => $area->bottom() - 1,
    VerticalAlignment::Top => $area->top(),
}
```

Becomes:

```php
match ($alignment) {
    VerticalAlignment::Bottom => $area->bottom() - 1,
    VerticalAlignment::Top    => $area->top(),
}
``` 

If you want, I can change this PR to include this one. If not, I'll only fix the Border class, manually. =D

[the remark this answers is not in the document]
Comment (dantleech) [reply src/Model/Modifier.php:20]: if you fix it manually will it not get reverted/fail cs?

[the remark this answers is not in the document]
Comment (KennedyTedesco) [reply src/Model/Modifier.php:20]: We'll need to stop using this rule and take care of those class constants manually.

**Comment by dantleech** [comment]
let's add the rule :)

**Pull request merged**